chart/k8gb/templates/external-dns/external-dns-ns1.yaml

```diff
@@ -15,6 +15,10 @@ spec:
         app: external-dns-ns1
     spec:
       serviceAccountName: external-dns
+      securityContext:
```

Comment:
How about making securityContext parameterized though `values.yaml` file, so it can be configured and reused in all external-dns deployments?
```suggestion
      securityContext:
{{- toYaml .Values.externaldns.securityContext | nindent 8 }}
```

Reply:
@somaritane good idea, implemented, please check